server/containers/genrepo/config/ssh/pub-packages-prod--root

```diff
@@ -0,0 +1,3 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA1yPOwMDXwSPXTdB3u7IEnRMHGz3q+YZpwYLB1yLY6fs8ya1kZ79Ic8vi5e/7MNffeZ87zXmcf+Os/JyJNShavQxhghSiMlZeuvgelMdXgUnjkW96NR5atAI8YeJ+X1Ir7sbS4TMaBMIGqLfnQhn37I8QMk+TEgWUSxiR/Om82vnlj+c9sgTE+iyvlHLo8tNAmwoaPESAc8QzxH2WPsVmmlKpLBbTt7Wv50qrSKSdOOZOcZrHuk+pT7ZoFyxFKWDkV+i8uDtnegNtbMSsTMRgPwjoQib0FBCAMQJxabrVyj60OdZ5/FYIbWIqdM8DrQHffU3iDU5x3WvVZDuBuZu1Dw==
```

Comment:
Are these supposed to be committed?  Switching keys would be more complicated this way.

Reply:
Right, I put them here before I made storage in the SecretsManager work. I'll remove that.